### ISSUE TITLE
Sonarr Alternate Title Not Parsing/Match Names Correctly

### ISSUE THREAD

**Issue by ExtremeFiretop** [Sonarr/Sonarr#6328]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

I download a French TV title like "La Petite Vie". It adds it to Sonarr as the English equivalent. "The little Life"
Sonarr however does detect "La Petite Vie" as a valid Alternate title as found below:

![image](https://github.com/Sonarr/Sonarr/assets/1971404/5558031e-9e16-4c39-b5ec-d5ec86f83ff1)

It adds it to my NZBGet Downloader correctly.  
However, after downloading is completed Sonarr rejects and blocklists the download saying it does not match the requested content (when it does). 

![image](https://github.com/Sonarr/Sonarr/assets/1971404/7b7a638d-875f-4a71-a1c9-e9585c5861c5)

The problem is Sonarr calls it "The little Life" instead of the French equivalent which was downloaded called "La Petite Vie"
Sonarr however does detect "La Petite Vie" as a valid alternative name as found below:

![image](https://github.com/Sonarr/Sonarr/assets/1971404/e78dc6a5-2c2d-4f5d-b842-de05c7997af4)

### Expected Behavior

For Alternate names to be valid when parsing and matching the downloaded items.

If Sonarr is monitoring a title called" The little Life"  and the downloaded title is "La Petite Vie" and that is detected as a valid alternate title, it should match as a valid alternate as found in the Sonarr profile.

### Steps To Reproduce

1. Make sure your "Quality profile" includes a "Custom Format" scoring the "Original Language" as the highest
![image](https://github.com/Sonarr/Sonarr/assets/1971404/0d835759-6551-4be8-8090-e3bf3dcfedb4)
2. Request Season 1 of "La Petite Vie" in French
3. It downloads it in the original French Language in NZBGet, but is quickly removed by Sonarr for not matching the "The little Life" name even though "La Petite Vie" is a valid Alternate name which started the search/queue
![image](https://github.com/Sonarr/Sonarr/assets/1971404/8505f2d4-0470-4e1b-b241-4e0e9fccbb9d)

### Environment

```markdown
- OS: Windows 11 23H2
- Sonarr: 4.0.0.748
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Edge
- Database: Sqlite 3.36.0
```

### What branch are you running?

Main

### Trace Logs?

[sonarr.debug.txt](https://github.com/Sonarr/Sonarr/files/13882230/sonarr.debug.txt)

**Comment by markus101** [comment]
The alias has 2023 and the release name does not. That’s not a bug, use the request form found in the FAQ to request the proper Alia’s using that release name.

Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.sonarr.tv/), or [IRC](https://web.libera.chat/?channels=#sonarr) for support/questions.

**Comment by ExtremeFiretop** [comment]
> The alias has 2023 and the release name does not. That’s not a bug, use the request form found in the FAQ to request the proper Alia’s using that release name.
> 
> Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.sonarr.tv/), or [IRC](https://web.libera.chat/?channels=#sonarr) for support/questions.

But why does it blacklist instead of holding it in the queue for manual import? I guess that's what Radarr does, maybe that's where I'm used to what behavior.

**Comment by markus101** [comment]
Sonarr doesn't block an item simply for being unmatched, I've heard people doing that with external tools, but it's not part of Sonarr.

**Comment by ExtremeFiretop** [comment]
> Sonarr doesn't block an item simply for being unmatched, I've heard people doing that with external tools, but it's not part of Sonarr.

Better response.. Thanks! :)
Is it blocking them though, so maybe that points me in the right direction.

is it possible it's a problem at the downloader instead? Since I think Sonarr does automatically block for a failed download. Correct? Will dive deeper into this. Thanks again!
